corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -0,0 +1,49 @@
+{% extends "style/bootstrap3/base_section.html" %}
+{% load hq_shared_tags %}
+{% load compress %}
+
+{% block stylesheets %}{{ block.super }}
+    {% compress css %}
+        <link type="text/css" rel="stylesheet" href="{% new_static 'app_manager/css/diff.css' %}"/>
+    {% endcompress %}
+{% endblock stylesheets %}
+
+{% block content %}
+<div class="container-fluid">
+    <h1>{{ first_app.name }}: Build #{{ first_app.version }} vs #{{ second_app.version }}</h1>
+    <p>See <a href="{% url "view_app" first_app.domain first_app.copy_of|default_if_none:first_app.get_id %}">original app</a> for the latest work in progress.</p>
+
+    <h2>Resource Files</h2>
+
+    <table class="table table-condensed">
+        {% for file_name, diff, adds, dels in diffs %}
+            <tr>
+                <td class="col-md-3">
+                    <a class="toggle-next" href="#">{{ file_name }}</a>
+                </td>
+                {# TODO: use simpler styling on additions and subtractions #}
+                <td class="col-md-9">
+                    <span class="diff"><span class="insert">+{{ adds }}</span></span>
+                    /
+                    <span class="diff"><span class="delete">-{{ dels }}</span></span>
+                </td>
+            </tr>
+            <tr>
+                <td colspan="2">
+                    {{ diff|safe }}
```

Comment:
Agreed. Updated to make files with no changes not clickable:
<img width="696" alt="screen shot 2015-09-22 at 11 01 04 am" src="https://cloud.githubusercontent.com/assets/2117127/10022265/d7b4d6f4-6119-11e5-90b5-9cd3cb23d882.png">